src/main/java/com/google/javascript/gents/CommentLinkingPass.java

```diff
@@ -126,7 +126,7 @@ private void addCommentToGroup(Node n) {
     /** Links a comment to the corresponding node */
     private void linkCommentToNode(Node n) {
       StringBuilder sb = new StringBuilder();
-      String sep = "";
+      String sep = "\n";
```

Comment:
Ah I see, the code review doesn't contain `comments.js` and only shows the changed parts there. Looks good.